assets/publish_lambda_layer.sh

```diff
@@ -3,6 +3,9 @@
 
 # First build layer
 layer=$(./assets/build_lambda_layer.sh)
+# Source the version from the previous script execution
+source ./assets/build_lambda_layer.sh
```

Comment:
this is a heavy script i would just copy the line with version or get it from version.json

Reply:
Replaced source with exact line which generates the version.